src/Microsoft.Identity.Client/Platforms/net45/NetDesktopPlatformProxy.cs

```diff
@@ -146,8 +146,13 @@ public override string GetBrokerOrRedirectUri(Uri redirectUri)
         }
 
         /// <inheritdoc />
-        public override string GetDefaultRedirectUri(string clientId)
+        public override string GetDefaultRedirectUri(string clientId, bool useRecommendedRedirectURI = false)
         {
+            if (useRecommendedRedirectURI)
+            {
+                return Constants.DefaultDesktopRedirectUri;
```

Comment:
Is this not relevant for macOS which is also a desktop platform?

Reply:
added it